main.css

```diff
@@ -0,0 +1,10 @@
+.maingrid {
+    display: grid;
+    grid-template: repeat(7, 1fr) / 150px, 350px, 150px;
+
+}
```

Comment:
Really love the idea to put the whole thing into a grid. Im excited to see how that ends up helping.